src/components/web-chat/WebChat.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div id="web-chat-container" v-if="webChatUrl && webChatReason">
+    <v-form
+      id="webchat"
+      target="webchat_window"
+      method="post"
+      :action="webChatUrl"
+      @submit="onSubmit()"
+    >
+      <input
+        type="hidden"
+        name="Reason"
+        :value="webChatReason"
+      />
+      <input
+        type="hidden"
+        name="UserLanguage"
+        value="en"
+      />
+      <input
+        type="hidden"
+        name="Parameters[TimeZoneOffset]"
+        :value="timeZoneOffset"
+      />
+
+      <v-tooltip top content-class="top-tooltip" nudge-top="5" :disabled="isMobile">
+        <template v-slot:activator="{ on, attrs }">
+          <div id="chat-button-wrapper" v-on="on" v-bind="attrs">
+            <v-btn
+              :disabled="(chatStatus !== 'open')"
+              large
+              outlined
+              color="bcgovblue"
+              elevation="4"
+              class="chat-button"
+              type="submit"
+              v-bind="attrs"
+              v-on="on"
+              aria-label="Chat with Helpdesk staff"
+            >
+              <v-icon class="mr-2 ml-n2">mdi-forum-outline</v-icon>
+              <span class="font-weight-bold">Chat</span>
+            </v-btn>
+          </div>
+        </template>
+        <span v-if="chatStatus === 'open'" id="open-tooltip-message">
+          Click here to chat live with Helpdesk staff about Name Requests.
+        </span>
+        <span v-else-if="chatStatus === 'closed'" id="closed-tooltip-message">
+          We are closed. The Service BC Contact Centre is open Monday through Friday
+          7:30am – 5:00pm Pacific Time excluding BC statutory holidays.
+        </span>
+        <span v-else id="unavailable-tooltip-message">
+          Webchat is temporarily unavailable.
+        </span>
+      </v-tooltip>
+    </v-form>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class ChatPopup extends Vue {
+  @Prop({ required: true }) readonly axios: any
+  @Prop({ default: false }) readonly isMobile: boolean
```

Comment:
This component is almost exactly the same as in Namerequest except for using these props instead of local axios and a getter.